packages/cloud-cognitive/carbon.yml

```diff
@@ -0,0 +1,613 @@
+    status: stable
+    type: pattern
+    platform: web
+    framework: react
+    thumbnailPath: './thumbnails/data-spreadsheet.svg'
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://carbon-for-ibm-products.netlify.app/?path=/story/ibm-products-components-dataspreadsheet-dataspreadsheet-canary--data-spreadsheet
+    tags:
+      - input-control
+  data-grid:
+    name: Data grid
+    status: stable
+    type: pattern
+    platform: web
+    framework: react
+    thumbnailPath: './thumbnails/data-grid.svg'
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://carbon-for-ibm-products.netlify.app/?path=/story/ibm-products-components-datagrid-datagrid-canary--basic-usage
+    tags:
+      - input-control
+      - content-block
+  edit-side-panel:
+    name: Edit side panel
+    status: stable
+    type: pattern
+    platform: web
+    framework: react
+    thumbnailPath: './thumbnails/edit-side-panel.svg'
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://carbon-for-ibm-products.netlify.app/?path=/story/ibm-products-patterns-edit-and-update-editsidepanel-canary--edit-side-panel
+    tags:
+      - input-control
+      - form
+  empty-state:
+    name: Empty state
+    status: stable
+    type: pattern
+    platform: web
+    framework: react
+    thumbnailPath: './thumbnails/empty-state.svg'
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://carbon-for-ibm-products.netlify.app/?path=/story/ibm-products-patterns-empty-state-emptystate
+    tags:
+      - content-element
+  example-component:
+    name: Example component
+    status: stable
+    type: component
+    platform: web
+    framework: react
+    thumbnailPath: './thumbnails/example-component.svg'
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://carbon-for-ibm-products.netlify.app/?path=/story/ibm-products-internal-examplecomponent--example-component
+    tags:
+      - input-control
+  export-modal:
+    name: Export modal
+    status: stable
+    type: pattern
+    platform: web
+    framework: react
+    thumbnailPath: './thumbnails/export-modal.svg'
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://carbon-for-ibm-products.netlify.app/?path=/story/ibm-products-patterns-export-exportmodal--with-success-message
+    tags:
+      - system-feedback
+  expressive-card:
+    name: Export modal
+    status: stable
+    type: component
+    platform: web
+    framework: react
+    thumbnailPath: './thumbnails/export-modal.svg'
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://carbon-for-ibm-products.netlify.app/?path=/story/ibm-products-patterns-export-exportmodal--with-success-message
```

Comment:
change link

Reply:
https://carbon-for-ibm-products.netlify.app/?path=/story/ibm-products-components-cards-expressivecard--default